docker/irods_client/tests/test_permissions.py

```diff
@@ -22,20 +22,14 @@ def test_permissions(session, item_name, request, tmpdir):
         upload(session, tmpdir/"bunny.rt.copy", ipath, overwrite=True)
     perm.set("own")
 
-    # TODO: Add more tests with other users.
-
-    # assert len(list(perm)) == 0
-    # ipath = IrodsPath(session, item.path)
-    # with pytest.raises(CAT_NO_ACCESS_PERMISSION):
-    #     download(session, ipath, tmpdir/"bunny.rt.copy", overwrite=True)
-    # perm.set("read", user=session.username, zone=session.zone)
-    # download(session, ipath, tmpdir/"bunny.rt.copy", overwrite=True)
-    # assert (tmpdir/"bunny.rt.copy").is_file
-    # with pytest.raises(ValueError):
-    #     upload(session, tmpdir/"bunny.rt.copy", ipath, overwrite=True)
-    # perm.set("write", user=session.username, zone=session.zone)
-    # # if item_name == "dataobject":
-    # upload(session, tmpdir/"bunny.rt.copy", ipath, overwrite=True)
-    # # else:
-    #     # upload(session, tmpdir/"bunny.rt.copy", ipath, overwrite=True)
-    # assert ipath.is_dataobject()
+    # Testing access for another user, only on plain irods
+    if not "set_home_perm" in config:
```

Comment:
This is probably not the right keyword for this test. Let's make a new keyword, e.g. `testuser_available` and set it to false for yoda. From now on, I think for these kind of keywords it's good to allow for the value not to be available, so we use `config.get(..., True)`. It would be nice to xfail on these tests, so perhaps it's better to split up this test function, where one is just the basic test and there is an advanced test that needs a testuser.

Reply:
Perfect!

Reply:
Done.